plugin/core/views.py

```diff
@@ -25,16 +25,14 @@ def get_line(window: Optional[sublime.Window], file_name: str, row: int) -> str:
 
 
 def point_to_offset(point: Point, view: sublime.View) -> int:
-    return view.text_point(
-        point.row,
-        # @see https://microsoft.github.io/language-server-protocol/specifications/specification-3-15/#position
-        # If the character value is greater than the line length it defaults back to the line length.
-        min(point.col, len(view.line(view.text_point(point.row, 0))))
-    )
+    # TODO: How can we clamp the column in UTF-16?
+    # @see https://microsoft.github.io/language-server-protocol/specifications/specification-3-15/#position
+    # If the character value is greater than the line length it defaults back to the line length.
+    return view.text_point_utf16(point.row, point.col)
```

Comment:
@jfcherng @tomv564 Unfortunately I couldn't find a way to clamp the column anymore in UTF-16. Perhaps we need to request a new ST API function for this.